_graphql/models.yml

```diff
@@ -0,0 +1,50 @@
+# Expose common static fields for the ElementEditor component to use for preview summaries
+CatchDesign\SS\TwigElemental\TwigElementalArea:
+  fields:
+    id: true
+    elements:
+      plugins:
+        paginateList: false
+  operations:
+    readOne: true
+
```

Comment:
needed to copy/paste the graph ql config from the elemental module, with an update to point to TwigElementalArea. to get the cms fetching for the content blocks working